README.md

```diff
@@ -95,6 +95,28 @@ $(document).ajaxComplete ->
   Twine.refresh()
 ```
 
+## Twine.register
+
+Optionally register a callback that will be called when the context that registers it is finished binding.
+
+Example:
+
+```html
+<div define="Twine.register(function() { console.log("done"); })"></div>
```

Comment:
This is a bad example. Define should return an object. Even if this is returning an object it's probably going to not do what you expect